TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/MenuHelper.php

```diff
@@ -64,8 +64,10 @@ public function buildSiteList(ControllerContext $controllerContext)
             $active = false;
             /** @var $site \TYPO3\Neos\Domain\Model\Site */
             if ($site->hasActiveDomains()) {
-                $hostname = $site->getFirstActiveDomain()->getHostPattern();
-                $active = $hostname === $requestUriHost;
+                $activeDomains = $site->getActiveDomains()->map(function ($domain) {
```

Comment:
<nitpick>wouldn't the result of this map be a list of `HostPattern`-strings? So maybe call this `$activeHostPatterns` instead?</nitpick>